traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/config/ConfigHandler.java

```diff
@@ -18,19 +18,11 @@
 import java.io.IOException;
 import java.net.UnknownHostException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeSet;
-import java.util.Iterator;
+import java.util.*;
```

Comment:
nit: I think intellij probably did this for you, but I think we pretty consistently try to use explicit imports of only what we use

Reply:
Using explicit imports in 9e680217f